Nrf pulse meas driver (#29248)
nrfconnect/sdk-nrf: samples/pulse_width_measurement/README.rst

.. _pulse_width_measurement_sample:

Pulse width measurement sample
##############################

> 

Reviewer: ```suggestion

.. contents::
   :local:
   :depth: 2
```

> Overview

Reviewer: Add "Overview" after "Requirements" section. Refer the sample template: https://ncsbmdoc.z6.web.core.windows.net/ncs/PR-29248/nrf/templates/sample_README.html#overview

********

> 

Reviewer 2: ```suggestion
```

> This sample demonstrates pulse width measurement using the pulse_meas driver.

Reviewer: Consider moving this line after the title.

A GPIOTE instance detects pulse edges, a TIMER instance measures the elapsed
time, and GPPI connects the peripherals without CPU intervention.

> Each reported value is a pulse width in microseconds.

Reviewer 2: ```suggestion
This sample demonstrates pulse width measurement using the ``pulse_meas`` driver.
A GPIOTE instance detects pulse edges, a TIMER instance measures the elapsed
time, and GPPI connects the peripherals without CPU intervention.

Each reported value represents the pulse width in microseconds.
```

Requirements
************

> * nRF54L15 DK or nRF54H20 DK

Reviewer 2: ```suggestion
This sample supports the following development kits:

.. table-from-sample-yaml::

```

> * A pulse source connected to the GPIO pins configured in the board overlay

Reviewer 2: ```suggestion
It also requires a pulse source connected to the GPIO pins configured in the board overlay.
```

Building and running
********************

.. |sample path| replace:: :file:`samples/pulse_width_measurement`

.. include:: /includes/build_and_run.txt

Supported targets:
   * ``nrf54l15dk/nrf54l15/cpuapp``

>    * ``nrf54h20dk/nrf54h20/cpuapp``

Reviewer 2: ```suggestion
```

These are in the sample's yaml, when the table is pulled in correctly in the previous section

Sample output
=============

> 

Reviewer: ```suggestion

The following output is displayed in the console:
```

.. code-block:: console

   Pulse width measurement sample start
   Stop requested, pending series: 0
   series[0]: first=1000 us last=1000 us

>    Measurement finished, series read: 1, callbacks: 1

Reviewer 2: ```suggestion
   Measurement finished, series read: 1, callbacks: 1
   
Dependencies
************
 
 ...
```
List the sample's dependencies in the Dependencies section.
